Storage.md

```diff
@@ -0,0 +1,230 @@
+This guide will describe how FullScreenShenanigans projects keep track of game information.
+
+### Table of Contents
+1. [ItemsHoldr](#itemsholdr)
+    1. [Local Storage](#local-storage)
+    2. [Items](#items)
+    3. [Updating](#updating)
+    4. [Clearing and Defaults](#clearing-and-defaults)
+    5. [Elements](#elements)
+2. [StateHoldr](#stateholdr)
+
+# ItemsHoldr
+
+ItemsHoldr is a versatile container to store and manipulate values in localStorage, and optionally keep an updated HTML container showing these values.
+
+## Local Storage
+
+`localStorage` is a browser property where information is saved without expiring.
+ItemsHoldr keeps track of items and their values while the game is running.
```

Comment:
I don't like these explanations of localStorage. Let's just assume the reader knows what it is, and say that ItemsHoldr stores things in it.